src/fr/guides/virtual-meetings/index.html

```diff
@@ -0,0 +1,387 @@
+		</ul>
+	</li>
+</ul>
+<h4>Ressource supplémentaire des États-Unis :</h4>
+<ul class="lst-spcd">
+	<li><a href="https://www.section508.gov/create/accessible-meetings/">Section 508.gov</a> de la GSA (en anglais).</li>
+</ul>
+
+<aside class="wb-fnote wb-init wb-fnote-inited" role="note" id="wb-auto-4"><h2 id="fn">Notes de bas de page</h2>
+	<dl>
+		<dt id="fn1-dt">Note de bas de page 1</dt>
+		<dd id="fn1">
+			<p class="fn-rtn"><a href="#fn1-rf"><span class="wb-inv">Revenir à la note de bas de page </span>1</a></p>
+			<p><a href="https://www.canada.ca/fr/gouvernement/fonctionpublique/mieux-etre-inclusion-diversite-fonction-publique/diversite-equite-matiere-emploi/accessibilite-fonction-publique/strategie-accessibilite-fonction-publique-tdm/strategie-accessibilite-fonction-publique-avant-propos.html">Avant-propos – Stratégie sur l’accessibilité au sein de la fonction publique du Canada</a></p>
+		</dd>
+		<dt id="fn2-dt">Note de bas de page 2</dt>
+		<dd id="fn2" tabindex="-1">
+			<p class="fn-rtn"><a href="#fn2-rf"><span class="wb-inv">Revenir à la note de bas de page </span>2</a></p>
+			<p><a href="https://www.canada.ca/fr/gouvernement/fonctionpublique/mieux-etre-inclusion-diversite-fonction-publique/diversite-equite-matiere-emploi/accessibilite-fonction-publique/strategie-accessibilite-fonction-publique-tdm/strategie-accessibilite-fonction-publique-introduction.html">Introduction: Stratégie sur l’accessibilité au sein de la fonction publique du Canada</a></p>
+		</dd>
+		<dt id="fn3-dt">Note de bas de page 3</dt>
+		<dd id="fn3" tabindex="-1">
+			<p class="fn-rtn"><a href="#fn3-rf"><span class="wb-inv">Revenir à la note de bas de page </span>3</a></p>
+			<p><a href="https://www1.nyc.gov/assets/mopd/downloads/pdf/virtual-meetings-accessibility-guide.pdf?msclkid=47052240af8611ecb816438bb9bcfb81">Bureau du maire de la ville de New York pour les personnes handicapées - Guide des réunions virtuelles accessibles (en anglais)</a></p>
+		</dd>
+		<dt id="fn4-dt">Note de bas de page 4</dt>
+		<dd id="fn4" tabindex="-1">
+			<p class="fn-rtn"><a href="#fn4-rf"><span class="wb-inv">Revenir à la note de bas de page </span>4</a></p>
+			<p><a href="https://www.gcpedia.gc.ca/gcwiki/images/1/14/Guide-Planifier_des_des_%C3%A9v%C3%A9nements_inclusifs_et_accessibles.pdf">Planification d’événements inclusifs et accessibles – Emploi et Développement social Canada (accessible seulement sur le réseau interne du GC)</a></p>
+		</dd>
+		<dt id="fn5-dt">Note de bas de page 5</dt>
+		<dd id="fn5" tabindex="-1">
+			<p class="fn-rtn"><a href="#fn5-rf"><span class="wb-inv">Revenir à la note de bas de page </span>5</a></p>
+			<p><a href="https://readabilityguidelines.co.uk/audiences-devices-channels/presentations/"><span lang="fr">Readability Guidelines – Content Design London (en anglais)</span></a></p>
+		</dd>
+		<dt id="fn6-dt">Note de bas de page 6</dt>
+		<dd id="fn6" tabindex="-1">
```

Comment:
remove the `tabindex="-1"` in the `dd` element

See [WET: Working Examples: Footnotes](https://wet-boew.github.io/v4.0-ci/demos/footnotes/footnotes-en.html) for more information.

Reply:
Done!